training/src/main/java/wolox/training/models/Book.java

```diff
@@ -18,6 +17,6 @@
     @Column(nullable = false) private String publisher;
     @Column(nullable = false) private String year;
     @Column(nullable = false) private int pages;
-    @Column(nullable = false) private String isbn;
+    @Column(nullable = false) @Id private String isbn;
```

Comment:
I highly recommend having a separate property for the id. Its a rare case but there are different books having the same ISBN, there shouldn't be because the ISBN is supposed to be unique, but there are some cases and it might bring you troubles later.
Friendly reminder: if you change this, don't forget to change the update and delete methods of the controller so they receive the boo's id instead of the ISBN